indico/htdocs/js/indico/widgets/person_link_widget.js

```diff
@@ -60,6 +60,12 @@
         var $buttonAlphaOrder = $fieldDisplay.find('.alpha-order-switch');
         var $form = $field.closest('form');
         var customOrder = !$buttonAlphaOrder.hasClass('active');
+        var maxManualPersonID = 0;
+
+        function setManualPersonID(person) {
+            person.id = 'manual-' + maxManualPersonID;
```

Comment:
There's no code on the server side that'll try to actually query a person/user with that ID?

And I'd s/manual/new/